docs/examples/howto/run_ckf_auto_tuning.rst

```diff
@@ -0,0 +1,98 @@
+ACTS Tutorial on Auto-Tuning in CombinatorialKalmanFilter (CKF)
+===============================================================
+The tracking algorithms require a number of pre-initialized parameters that are often hand-tuned to obtain high performance. Usually, the value of these parameters change as the underlying geometrical or magnetic configuration changes. An automatic tuning of these parameters can be very useful for obtaining highly efficient parameter configuration as well as for studying different detector geometries. This tutorial is based on parameter optimization studies using two different optimization frameworks: Optuna and Orion. Eight parameters of Track Seeding algorithm have been tuned using these frameworks and their performance have been studied on CKF. 
+
+Prerequisites
+-------------
+Since Optuna and Orion are independent frameworks, these need to be installed separately. The following setup works on LXPLUS that requires creating a virtual python environment:
```

Comment:
Instead of LXPLUS you could say something like "any machine with access to cvmfs" (I use the same setup on my lab local server)